packages/react/src/components/Storage/FileUploader/Previewer/__tests__/Previewer.test.tsx

```diff
@@ -27,82 +27,144 @@ const fileStatus2: FileStatus = {
 };
 const fileStatuses: FileStatuses = [fileStatus];
 const fileStatuses2: FileStatuses = [fileStatus, fileStatus2];
+const ref = {
+  current: {
+    info: jest.fn(),
+  },
+} as unknown as React.MutableRefObject<HTMLInputElement>;
+
+const commonProps = {
+  acceptedFileTypes: ['.png'],
+  fileStatuses: fileStatuses,
+  inDropZone: false,
+  isEditingName: [],
+  isLoading: false,
+  isSuccess: false,
+  maxFilesError: false,
+  multiple: true,
+  onClear: () => null,
+  onDragEnter: () => null,
+  onDragLeave: () => null,
+  onDragOver: () => null,
+  onDragStart: () => null,
+  onDrop: () => null,
+  onFileChange: () => null,
+  onFileClick: () => null,
+  percentage: 0,
+  hiddenInput: ref,
+  onUploadButtonClick: () => null,
+};
 
 describe('Previewer', () => {
   it('renders as expected', async () => {
-    const { container } = render(
-      <Previewer
-        acceptedFileTypes={['.png']}
-        fileStatuses={fileStatuses}
-        inDropZone={false}
-        isEditingName={[]}
-        isLoading={false}
-        isSuccess={false}
-        maxFilesError={false}
-        multiple={true}
-        onClear={() => {}}
-        onDragEnter={() => {}}
-        onDragLeave={() => {}}
-        onDragOver={() => {}}
-        onDragStart={() => {}}
-        onDrop={() => {}}
-        onFileChange={() => {}}
-        onFileClick={() => {}}
-        percentage={0}
-      />
-    );
+    const { container } = render(<Previewer {...commonProps} />);
 
     expect(container).toMatchSnapshot();
   });
   it('has two files passed in shows 2 files selected', async () => {
     const { findByText } = render(
-      <Previewer
-        acceptedFileTypes={['.png']}
-        fileStatuses={fileStatuses2}
-        inDropZone={false}
-        isEditingName={[]}
-        isLoading={false}
-        isSuccess={false}
-        maxFilesError={false}
-        multiple={true}
-        onClear={() => {}}
-        onDragEnter={() => {}}
-        onDragLeave={() => {}}
-        onDragOver={() => {}}
-        onDragStart={() => {}}
-        onDrop={() => {}}
-        onFileChange={() => {}}
-        onFileClick={() => {}}
-        percentage={0}
-      />
+      <Previewer {...commonProps} fileStatuses={fileStatuses2} />
     );
 
     expect(await findByText('2 files selected')).toBeVisible();
   });
 
   it('has one file passed in and shows one selected ', async () => {
+    const { findByText } = render(
+      <Previewer {...commonProps} fileStatuses={fileStatuses} />
+    );
+
+    expect(await findByText('1 files selected')).toBeVisible();
```

Comment:
Nit: prefer using `screen.findByText`, to match other React unit tests:

```
import { screen } from '@testing-library/react

  // ...
  expect(await screen.findByText(...))
```

Reply:
Same with other unit tests below

Reply:
Done in f3d3571